lib/new.js

```diff
@@ -43,6 +43,19 @@ function getPackageName(name) {
   return `skyux-${prefix}-${name}`;
 }
 
+/**
+ * Returns a string to be used for the description property in package.json/logger.
+ * @name getPackageDescriptionType
+ */
+function getPackageDescriptionType(settings) {
+  switch (settings.template.name) {
+    case 'library':
+      return 'Library';
```

Comment:
This is pedantic, and more of a question: could we print these in lower-case, or is it proper to make them title-case?

Reply:
In one use-case it's the beginning of the sentence.  In another, it's in the middle.  Would you prefer I use the correct case for both.  That seems a little like overkill, but I do want it to be grammatically correct.

I supposed I could make it always the beginning or not and then having a single casing would be appropriate.

Reply:
It's not a huge deal; just something that seemed jarring to me personally. We can always come back to it if needed.

Reply:
I'd like to go ahead and fix it now that you've pointed it out.  :-)

Any suggestions on how to word the two sentences such that both make sense and that "SPA" or "Library" is first (so it can be capitalized).

Reply:
Sorry, meant to tag @blackbaud-johnly as well.  :-)